crates/ruff/src/rules/pandas_vet/rules/inplace_argument.rs

```diff
@@ -8,6 +8,26 @@ use crate::rules::pandas_vet::fixes::fix_inplace_argument;
 use crate::violation::AlwaysAutofixableViolation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for `inplace=True` in code using the `pandas` library.
+    /// 
+    /// ## Why is this bad?
+    /// - Many people expect `inplace=True` to be a performance benefit that prevents dataframe copies, but that's often not true.
+    /// - It encourages mutation rather than immutable data, which is harder to reason about and may cause bugs.
+    /// - It removes the ability to use the chaining style for `pandas` code.
```

Comment:
nit: we may want to explicitly mention `method chaining` instead of just `chaining`

Reply:
done